demos/common/cpp_gapi/utils_gapi/include/utils_gapi/gapi_features.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <opencv2/core/version.hpp>
+#include <opencv2/gapi/core.hpp>
+
+#if CV_VERSION_MAJOR >= 4
+    #define GAPI_IE_BACKEND
+#endif
+
+#if CV_VERSION_MAJOR >= 4 && CV_VERSION_MINOR >= 7
+    #define GAPI_ONNX_BACKEND
```

Comment:
The presence of ONNX backend depends not only on OpenCV version but also on cmake flags used to compile OpenCV. Can you check if OpenCV already provides a define which could indicate that?

Reply:
Inference backend (a.k.a `ie`/`ov`/`onnx` params) exist always regardless backend availability.
In case `OpenCV` isn't compiled with proper inference backend it will throw the error in runtime